internal/command/launch/cmd.go

```diff
@@ -193,6 +208,9 @@ func run(ctx context.Context) (err error) {
 			return err
 		}
 	}
+	if requireUi {
```

Comment:
not completely following the logic but shouldn't it check err ?

Reply:
yup, that's a bug!

also, this
> not completely following the logic

is probably a sign that it needed a tad of cleanup :)